### ISSUE TITLE
In-memory adapter?

### ISSUE THREAD

**Issue by unruledboy** [mrahhal/MR.AspNetCore.Jobs#17]
There are cases we simply want the jobs to be in memory only, not being backed by databases.

Can we just hold the jobs in memory?

Thanks.

**Comment by mrahhal** [comment]
Good idea. There's no in memory adapter right now, but implementing it shouldn't be very hard.
